File: Data/Models/Comentario.cs
namespace TrabalhoAPI.Data.Models
{
    public class Comentario
    {
        public int Id { get; set; }
        public string Texto { get; set; }
        public int UsuarioId { get; set; }
        public Usuario? Usuario { get; set; }
        public int FilmeId { get; set; }
        public Filme? Filme { get; set; }
    }
}

File: src/app/Comentario.ts
import { Filme } from "./Filme";
import { Usuario } from "./Usuario";

export class Comentario{
    id: number = 0;
    texto: string = '';
    usuarioId: number = 0;
    usuario: Usuario | null = null;
    filmeId: number = 0;
    filme: Filme | null = null;
}